src/Toggle/Entity/ActivatableToggle.php

```diff
@@ -0,0 +1,8 @@
+<?php
+namespace Clearbooks\Labs\Toggle\Entity;
+
+interface ActivatableToggle
```

Comment:
Yep. I'm still trying to wrap my head around whether that should be stored or calculated on the fly.

I think perhaps it should be stored.

@viktorcb what's your opinion